pkg/deployment/config.go

```diff
@@ -62,11 +62,11 @@ const (
 	queueSidecarMemoryLimitKey           = "queueSidecarMemoryLimit"
 	queueSidecarEphemeralStorageLimitKey = "queueSidecarEphemeralStorageLimit"
 
-	// queueOnNoMoreTrafficEndpointKey is the key to configure the endpoint for acting when traffic drops to / increases from zero.
-	queueOnNoMoreTrafficEndpointKey = "queueOnNoMoreTrafficEndpoint"
+	// concurrencyStateEndpointKey is the key to configure the endpoint for acting when traffic drops to / increases from zero.
+	concurrencyStateEndpointKey = "concurrencyStateEndpoint"
 
-	// queueOnNoMoreTrafficEndpointDefault is the default endpoint for acting when traffic drops to / increases from zero.
-	queueOnNoMoreTrafficEndpointDefault = ""
+	// ConcurrencyStateEndpointDefault is the default endpoint for acting when traffic drops to / increases from zero.
+	ConcurrencyStateEndpointDefault = ""
```

Comment:
Totally a nit but I don't think it's really worth having this as a public constant vs. just checking against the string being empty everywhere. I feel like empty string checks would be clearer.